.github/workflows/update-deployment-ee.yml

```diff
@@ -0,0 +1,66 @@
+name: "2. (Internal) Update Deployment - EE"
+# For Partner Team EE to deploy updates to their application
+run-name: "${{inputs.target_env}}: Update deployment domain-ee ${{inputs.image_tag}}"
+
+on:
+  workflow_dispatch:
+    inputs:
+      target_env:
+        description: 'Target LHDI environment'
+        required: true
+        default: 'dev'
+        type: choice
+        # Partner Teams are not allowed to deploy to prod
+        options:
+        - dev
+        - qa
+        - sandbox
+        - prod-test
+
+      image_tag:
+        description: 'Image tag: first 7 of commit hash, "latest", "v1.2.3", or "release-1.2.3"'
+        required: true
+        default: 'latest'
+        type: string
+
+      # Other deployment options
+      rollback:
+        description: 'If deploy fails, rollback to previous? (Always true for prod)'
+        required: true
+        type: boolean
+        default: false
+      shutdown:
+        description: '(As a last resort) Shut down chart completely, then redeploy'
+        required: true
+        type: boolean
+        default: false
+
+  # Allow other workflows to call this one
+  workflow_call:
+    inputs:
+      target_env:
+        required: true
+        type: string
+      image_tag:
+        required: true
+        type: string
+
+      rollback:
+        required: false
+        type: boolean
+        default: false
+      shutdown:
+        required: false
+        type: boolean
+        default: false
+
+jobs:
+  update-cc-deployment:
+    uses: ./.github/workflows/update-deployment.yml
+    secrets: inherit
+    with:
+      target_env: ${{ inputs.target_env }}
+      helm_chart: domain-ee
```

Comment:
Need to update https://github.com/department-of-veterans-affairs/abd-vro/blob/d64aaf1ddfa0649dc97c09f88ddd49beac5cd217/.github/workflows/update-deployment.yml#L23-L25

Reply:
`update-deployment-ee.yml` does not need to be changed. I was trying to convey that since you refer to `domain-ee` in `update-deployment-ee.yml`, you need to update `update-deployment.yml`.

Reply:
I see entry in domain-ee in update-deployment.yaml. is this enough or you think it still need to be modified?

Reply:
You added [that entry](https://github.com/department-of-veterans-affairs/abd-vro/pull/1922/files#diff-f7f98f727b0f49e39967a55e5be72e319c312412f045d91c32b9d35b327134c1R30). It's enough.